product_docs/docs/pgd/5/known_issues.mdx

```diff
@@ -2,56 +2,50 @@
 title: 'Known issues'
 ---
 
-This section discusses currently known issues in EDB Postgres Distributed 5. 
-These known issues are tracked in PGD's
-ticketing system and are expected to be resolved in a future
-release.
-
--   If the resolver for the `update_origin_change` conflict
-    is set to `skip`, `synchronous_commit=remote_apply` is used, and
-    concurrent updates of the same row are repeatedly applied on two
-    different nodes, then one of the update statements might hang due
-    to a deadlock with the PGD writer.  As mentioned in the
-    [Conflicts](consistency/conflicts/) chapter, `skip` is not the default
-    resolver for the `update_origin_change` conflict, and this
-    combination isn't intended to be used in production. It discards
-    one of the two conflicting updates based on the order of arrival
-    on that node, which is likely to cause a divergent cluster.
-    In the rare situation that you do choose to use the `skip`
-    conflict resolver,  note the issue with the use of the
-    `remote_apply` mode.
-
--   The Decoding Worker feature doesn't work with CAMO/EAGER/Group Commit. Installations using CAMO/Eager/Group Commit must keep `enable_wal_decoder` disabled.
+This section discusses currently known issues in EDB Postgres Distributed 5.
+These known issues are tracked in PGD's ticketing system and are expected to be
+resolved in a future release.
+
+-   The Decoding Worker feature doesn't work with CAMO/EAGER/Group Commit.
+    Installations using CAMO/Eager/Group Commit must keep `enable_wal_decoder`
+    disabled.
 
 -   Lag control doesn't adjust commit delay in any way on a fully
     isolated node, that is, in case all other nodes are unreachable or not
     operational. As soon as at least one node is connected, replication
     lag control picks up its work and adjusts the PGD commit delay
     again.
 
--   For time-based lag control, PGD currently uses the lag time (measured
-    by commit timestamps) rather than the estimated catchup time that's
-    based on historic apply rate.
-
--   Changing the CAMO partners in a CAMO pair isn't currently possible.
-    It's possible only to add or remove a pair.
-    Adding or removing a pair doesn't need a restart of Postgres or even a
-    reload of the configuration.
```

Comment:
That's good to remove, we don't plan to "drop" or "fix" this limitation.

Reply:
The question is should it be moved to Limitations though? For Limitations it could just read:

 **Changing the CAMO partners in a CAMO pair isn't possible; you can only add or remove a pair.**

Reply:
Added to limitations.